src/components/popover/popover.tsx

```diff
@@ -80,6 +80,9 @@ export class Popover {
     @Prop({ reflect: true })
     public openDirection: OpenDirection;
 
+    @Prop()
+    public autoFocus = false;
```

Comment:
This won't conflict with `autofocus` since the camel-casing will turn this into an attribute named `auto-focus`. But it's probably confusing anyway to name it like that :smile:

Reply:
I agree that using something like the `dialog` element is probably better. I suggested we add a prop for this for now so that we do not cause any breaking behavior. But maybe we don't need the prop, maybe we can check the global `autofocus` attribute instead

Reply:
> But maybe we don't need the prop, maybe we can check the global autofocus attribute instead

I guess we can query for an element matching `[autofocus]`  and give that focus when showing the popover. That's how I understand things would work when using https://developer.mozilla.org/en-US/docs/Web/API/HTMLDialogElement/showModal

We would then have a bit more control over which element is given focus, rather than just relying on `delegateFocus` when we're using shadow dom for the content.